jquery.flot.composeImages.js

```diff
@@ -103,18 +103,44 @@
     }
 
     function copySVGToImgMostBrowsers(svg, img) {
+        //get svg source.
         var rules = getCSSRules(document),
-            text = embedCSSRulesInSVG(rules, svg),
-            blob = new Blob([text], {type: "image/svg+xml;charset=utf-8"}),
+            source = embedCSSRulesInSVG(rules, svg);
+
+        //add name spaces.
+        if (!source.match(/^<svg[^>]+xmlns="http:\/\/www\.w3\.org\/2000\/svg"/)) {
```

Comment:
Could you please move these 3 fixes in a different function **patchSVGSource(source)** and call it from both locations.